docs/topic_guides/comparison_templates.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Out-of-the-box Comparisons for specific data types\n",
+    "\n",
+    "Similarity is defined differently for types of data (e.g. names, dates of birth, postcodes, addresses, ids). The [Comparison Template Library](customising_comparisons.ipynb#method-2-using-the-comparisontemplatelibrary) contains functions to generate read-made comparisons for a variety of data types.\n",
+    "\n",
+    "Below are examples of how to structure comparisons for a variety of data types."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Date Comparisons\n",
+    "\n",
+    "Date comparisons are generally structured as: \n",
+    "\n",
+    "- Null level  \n",
+    "- Exact match  \n",
+    "- Fuzzy match ([using metric of choice](comparators.md))  \n",
+    "- Interval match (within X days/months/years)  \n",
+    "- Else level\n",
+    "\n",
+    "The [comparison_template_library](#method-2-using-the-comparisontemplatelibrary) contains the `date_comparison` function which gives this structure, with some pre-defined parameters, out-of-the-box."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
Add a markdown `<hr>` break between each section to more clearly differentiate between each part?